official/docs/ruby/customs_info/create_customs_info.rb

```diff
@@ -1,26 +1,24 @@
 require 'easypost'
 
-EasyPost.api_key = ENV['EASYPOST_API_KEY']
+EasyPost.api_key = ENV['EASYPOST_TEST_API_KEY']
```

Comment:
```suggestion
EasyPost.api_key = ENV['EASYPOST_API_KEY']
```
Why was this changed?

Reply:
Ah dang, left over from a test. I'll fix this in the next PR so we don't have to get stamps again.